<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>419</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Comic Sans MS</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Clock</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::DocumentOpenRecent"/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>291</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::TextElideMode::ElideMiddle</enum>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>World Clock</string>
     </attribute>
     <widget class="QLabel" name="label_iran">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Iran</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_german">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Germany</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_usa">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>13</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>USA</string>
      </property>
     </widget>
     <widget class="QLabel" name="germany_label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>280</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="usa_label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="iran_label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>29</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Alarm</string>
     </attribute>
     <widget class="QPushButton" name="add_alarm">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>300</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>11</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="time">
       <time>
        <hour>7</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>261</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gird_alarm"/>
     </widget>
    </widget>
    <widget class="QWidget" name="Stopwatch">
     <attribute name="title">
      <string>Stopwatch</string>
     </attribute>
     <widget class="QPushButton" name="start_stopwatch">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>330</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stop_stopwatch">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>330</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_stopwatch">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>261</width>
        <height>311</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Seven Segment</family>
        <pointsize>35</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="reset_stopwatch">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>330</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Timer</string>
     </attribute>
     <widget class="QLineEdit" name="txt_min">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Seven Segment</family>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>15</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_sec">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Seven Segment</family>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>30</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_hour">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Seven Segment</family>
        <pointsize>25</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>00</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="stop_timer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start_timer">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>330</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reset_timer">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>330</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Print</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>306</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
